extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -73,7 +73,7 @@
   "comscore": {
     "host": "https://sb.scorecardresearch.com",
     "base": "https://sb.scorecardresearch.com/b?",
-    "pageview": "https://sb.scorecardresearch.com/b?c1=2&c2=1000001&cs_ucfr=&cs_amp_consent=&cs_pv=_page_view_id_&c12=_client_id_&rn=_random_&c8=_title_&c7=_canonical_url_&c9=_document_referrer_&cs_c7amp=_ampdoc_url_"
+    "pageview": "https://sb.scorecardresearch.com/b?c1=2&c2=1000001&cs_ucfr=_IF__IF__IF_&cs_amp_consent=_CONSENT_STATE_&cs_pv=_page_view_id_&c12=_client_id_&rn=_random_&c8=_title_&c7=_canonical_url_&c9=_document_referrer_&cs_c7amp=_ampdoc_url_"
```

Comment:
Just realize that the change here would make it difficult to revert change if we want to unlaunch the json vendor config change. 
One option is to change the test-vendor.js as well, another option is to wait for another 2 weeks. What do you think?